### DISCUSSION TITLE
What is the new icon top left?

### DISCUSSION THREAD

**Discussion by CalbertAmu** [koreader/koreader#10220] [Q&A] [answered]
It appears when I toggle something from the bottom menu, like margins, or Alt Status Bar. It does a cute `animation`

KOBO A1, newest beta.

![Reader_2023-03-18_123602](https://user-images.githubusercontent.com/16507083/226121195-4895eb08-abde-4f25-86f2-bb64aced4342.png)
![Reader_2023-03-18_124214](https://user-images.githubusercontent.com/16507083/226121196-ff42fdf1-8dce-40d3-93a7-81ad74dee2ae.png)

**Comment by poire-z** [comment, marked as answer]
> It does a _cute_ animation

Thank you :) We had to gather many international artists to achieve that cuteness, discussions started at https://github.com/koreader/koreader/issues/9087#issuecomment-1416743867.
It was implemented by https://github.com/koreader/koreader/pull/10124.
BUT DO NOT GO FOLLOWING THESE LINKS YET !

Before going there, look at this screenshot, and see if you can understand/figure out yourself what the animation (and the steps) means - and tell us:

![image](https://user-images.githubusercontent.com/24273478/226122103-a052a874-fba1-4810-9169-a4cf784d7cb1.png)

It's new, I'd like to know how obvious or not these icons may be, and if not, how much we should advertise that.

**Reply by CalbertAmu** [reply]
First I thought it might be a 'loading' icon, but I didn't know why we'd need that. So I was confused. And I didn't think e-readers needed to 'pre-load' the upcoming pages, that it happened real-time, as pages were turned.

Thanks!
